src/components/FeaturedTreesSlider/index.js

```diff
@@ -16,39 +18,42 @@ function FeaturedTreesSlider({ trees, size = null }) {
   // if size="small" props is passed in, size of images= 144px
   const { classes } = useStyles(size);
   const sliderRef = useRef();
+  const isMobile = useMediaQuery(useTheme().breakpoints.down('sm'));
```

Comment:
@Mohmn I suggest letting the parent component decide if we need show the buttons, so the component gets relief of awaring of the knowledge of the device media. IMO all the OOP is about who should know what, the parent already knows the media info, so let them tell the component to display or hide the buttons.

Reply:
updated @dadiorchen